packages/itwinui-css/scripts/lightningCssSettings.js

```diff
@@ -3,9 +3,9 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 const targets = {
-  chrome: (90 << 16) | (0 << 8), // chrome 90.0
-  firefox: (90 << 16) | (0 << 8), // firefox 90.0
-  safari: (14 << 16) | (0 << 8), // safari 14.0
+  chrome: (105 << 16) | (0 << 8), // chrome 105.0
```

Comment:
I think this needs to be mentioned in our docs (or migration guide) that this package works best with these browser versions.

Reply:
it's interesting, because we don't really target specific browser versions on purpose. maybe we could mention a moving target like "it works with last two major versions of evergreen browsers". see example: https://shoelace.style/index.html#browser-support

Reply:
"Design for ..." or "Best experience with ..."